queries.sql

```diff
@@ -40,3 +40,61 @@ SELECT *
 FROM animals
 WHERE weight>=10.4 and weight<=17.3 
 
+/**ex-2*/
+/*Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.*/
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon'
+
+/*Update the animals table by setting the species column to pokemon for all animals that don't have species already set*/
+UPDATE animals
+SET species="pokemon"
+WHERE species IS NULL;
+
+/* delete all records in the animals table, then roll back the transaction.*/
+DELETE FROM animals
```

Comment:
- [ ] Inside a transaction you should delete all records in the animal's table, then roll back the transaction, It may look like this :point_down: 

```sql
BEGIN;

   DELETE FROM animals WHERE ;
   SELECT COUNT(*) FROM ANIMALS;

   ROLLBACK;
   SELECT COUNT(*) FROM ANIMALS;
```